server/data/gotenbergClient.ts

```diff
@@ -1,25 +1,10 @@
 import superagent from 'superagent'
 import logger from '../../logger'
 import config from '../config'
-
-export type PdfOptions = {
```

Comment:
I have moved the `PdfOptions` type to the PDF Renderer rather than this gotenberg client class - on the basis that the gotenberg client isnt really anything to with PDFs per.se - its really just a REST client class that happens to know about the specifics of calling the gotenberg container.